src/clients/aa-genprof.rb

```diff
@@ -0,0 +1,5 @@
+require 'apparmor/genprof.rb'
+
+gp = AppArmor::GenProf.new
+gp.execute unless gp.nil?
```

Comment:
new never return nil, it always return instance of class ( unless you redefine it, which is really really bad approach )